tests/cibyl/unit/cli/test_query.py

```diff
@@ -183,7 +181,7 @@ def test_get_tests_builds(self):
             'last_build': None
         }
 
-        self.assertEqual(QueryType.TESTS, get_query_type(**args))
+        self.assertIn(QueryType.TESTS, get_query_type(**args))
 
 
 class TestGetQueryTypeOpenstackPlugin(OpenstackPluginWithJobSystem):
```

Comment:
There a few tests in this class that are now failing because the behaviour of the QuerySelector changed. They combine openstack arguments with `--tenant`, and before would ignore the tenant part. Now, they should read both.

Reply:
Fixed.